src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala

```diff
@@ -401,11 +406,22 @@ class FileServiceSpec extends AnyFlatSpec with MockitoSugar with Matchers with S
         case _ => count should equal(2)
       }
     })
+    verify(fileStatusRepositoryMock, times(2)).addFileStatuses(any[List[FilestatusRow]]())
```

Comment:
This is a duplicate of the verify call on line 379. You can get rid of one of them.

Reply:
Removed. Thanks for highlighting it, probably it's a copy-paste error